backend/protocol_rpc/endpoints.py

```diff
@@ -52,38 +52,31 @@ def clear_db_tables(db_client: DBClient, tables: list) -> dict:
 
 
 ####### ACCOUNTS ENDPOINTS #######
-def create_account(accounts_manager: AccountsManager) -> dict:
-    new_account = accounts_manager.create_new_account(0)
-    return {"account_address": new_account.address}
-
-
-def fund_account(
-    accounts_manager: AccountsManager, account_address: str, amount: int
-) -> dict:
+def get_balance(accounts_manager: AccountsManager, account_address: str) -> dict:
     if not accounts_manager.is_valid_address(account_address):
-        raise InvalidAddressError(account_address)
-
-    accounts_manager.fund_account(account_address, amount)
-    return {"account_address": account_address, "amount": amount}
+        raise InvalidAddressError(
+            account_address, f"Invalid address from_address: {account_address}"
+        )
+    account_balance = accounts_manager.get_account_balance(account_address)
+    return {"account_balance": account_balance}
 
 
-def send_transaction(
-    transactions_processor: TransactionsProcessor,
+def fund_account(
     accounts_manager: AccountsManager,
-    from_account: str,
-    to_account: str,
+    transactions_processor: TransactionsProcessor,
+    account_address: str,
     amount: int,
 ) -> dict:
-    if not accounts_manager.is_valid_address(from_account):
-        raise InvalidAddressError(from_account)
+    if not accounts_manager.is_valid_address(account_address):
+        raise InvalidAddressError(account_address)
 
-    if not accounts_manager.is_valid_address(to_account):
-        raise InvalidAddressError(to_account)
+    account = accounts_manager.get_account(account_address)
+    if not account:
+        accounts_manager.create_new_account_with_address(account_address)
 
```

Comment:
I think this logic is redundant since it's already handled at `update_account_balance`, which should be called when processing the transaction

Reply:
This is just about the format, not about the account existing. Now, in the `update_account_balance` if the account doesn't exists, it gets created since that's the Ethereum behaviour

Reply:
@cristiam86 I see this logic

```py
    account = accounts_manager.get_account(account_address)
    if not account:
        accounts_manager.create_new_account_with_address(account_address)
```
Which to me seems duplicated in `update_account_balance`

Reply:
You are right, probably due to the refactor. I left it in the update_account_balance which is called from the transfer as well